website/src/components/Changelog/Gateway.tsx

```diff
@@ -8,6 +8,14 @@ export default function Gateway() {
 
   return (
     <Entries href={href} arches={arches} title="Gateway">
+      <Entry version="1.1.X" date={new Date("2024-XX-XX")}>
+        <ul className="list-disc space-y-2 pl-4 mb-4">
+          <li className="pl-2">
+            Remove `FIREZONE_ENABLE_MASQUERADE` env variable.
```

Comment:
```suggestion
            Removes `FIREZONE_ENABLE_MASQUERADE` env variable.
```

Nit: In the Client changelogs, we've been phrasing it as "(Version 1.1.x) removes some removed features"